File: golang/definitions/precision.go
// Calculates the default precision of a binary floating point implementation.
func precision() float64 {
	n := 0.0
	x := 1.0
	for 1.0 != 1.0+x {
		x /= 2.0
		n += 1
	}
	return n
}

File: javascript/definitions/precision.js
// precision() -> number
// Calculates the default precision of a binary floating point implementation.
// precision() -> 53 [env: Firefox]

function precision() {
    let n = 0;
    let x = 1;
    while (1 !== 1 + x) {
        x /= 2;
        n += 1;
    }
    return n;
}
